.github/workflows/nightly-rainforest.yml

```diff
@@ -0,0 +1,122 @@
+name: nightly-rainforest
+
+on:
+  schedule:
+    - cron: 0 4 * * 0-5
+
+defaults:
+  run:
+    shell: bash
```

Comment:
This can be removed, since bash is the default shell for linux runners in ghactions